### PR TITLE
Syntax highlight more languages

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/8955

### CODE REVIEW

**Comment by sethladd** [comment]
cc @mit-mit

File: _sass/_prism.scss [hunk 1]
```diff
@@ -97,41 +97,45 @@ pre[class*="language-"] {
...
 }
 
 .token.operator,
 .token.entity,
 .token.url,
 .language-css .token.string,
 .style .token.string {
-  color: #a67f59;
+	color: #a67f59;
+	background: hsla(0, 0%, 100%, .5);
```
Comment (mit-mit) [inline _sass/_prism.scss]: Just checking: Did you mean to add this?

File: _sass/_prism.scss [hunk 2]
```diff
@@ -97,41 +97,45 @@ pre[class*="language-"] {
...
-  font-style: italic;
+	font-style: italic;
 }
 
 .token.entity {
-  cursor: help;
+	cursor: help;
 }
+
+DOWNLOAD CSS
```
Comment (mit-mit) [inline _sass/_prism.scss]: Jekyll doesn't like these last two lines:

```
 Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
                    Invalid CSS after "... awesome people": expected "{", was "" on line 142
```

File: _sass/_prism.scss (hunk 1 above)
Comment (sethladd) [inline _sass/_prism.scss]: I meant to update this file holistically, as it's generated from prism.

File: _sass/_prism.scss (hunk 2 above)
Comment (sethladd) [inline _sass/_prism.scss]: Yikes! I copied too much. Thanks for the catch

**Comment by sethladd** [comment]
@mit-mit ptal

**Comment by mit-mit** [comment]
I still can't get this to work. Neither ` ```java `, nor `{% prettify java %}` seem to actually produce any formatting? I do see some tabs being removed, but no colors.

**Comment by sethladd** [comment]
Hm, sorry for the trouble. I'll give it a shot and see what's up.

**Comment by sethladd** [comment]
Fixed! Sorry, copied too much.

screen shot 2017-03-28 at 10 40 18 am
screen shot 2017-03-28 at 10 40 14 am

**Comment by sethladd** [comment]
@mit-mit ptal

**Pull request merged**